api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -168,7 +167,7 @@ public void setUp() {
   public void countParticipantsRaiseDeadlineExceededException() {
     stubBigQueryCallThrowDeadlineExceededException();
     assertThrows(
-        DeadlineExceededException.class,
+        RuntimeException.class,
```

Comment:
Type of exception doesn't matter here since it's just what we're throwing below. However, I think this is an improvement over the previous since it makes the value of the test more obvious.